libs/blocks/marquee/marquee.css

```diff
@@ -296,6 +304,22 @@
     display: block;
   }
 
+  .marquee.large.tablet-dark {
+    background-color: black;
+  }
+
+  .marquee.large.tablet-light {
+    background-color: initial;
+  }
+
+  .marquee.tablet-light {
+    color: var(--text-color);
+  }
+
+  .marquee.tablet-dark {
+    color: #FFF;
```

Comment:
I see hex, variables and named color values are used interchangeably, it might not be a bad idea to standardise and stick to a rule. If an update is needed, it will be easier to check for just `#fff`, `var(--whatever)` or `white` instead of all of them.